Octokit/Clients/IRepositoryPagesClient.cs

```diff
@@ -33,6 +34,19 @@ public interface IRepositoryPagesClient
         /// </remarks>
         /// <returns></returns>
         Task<IReadOnlyList<PagesBuild>> GetAll(string owner, string repositoryName);
+
+        /// <summary>
+        /// Gets all build metadata for a given repository
+        /// </summary>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="repositoryName">The name of the repository</param>
+        /// <param name="options">Options for changing the behaviour of the API</param>
+        ///  <remarks>
```

Comment:
extraneous whitespace here? between /// and <remarks> there are 2 spaces instead of 1.  Seems to be the case on a few of the xmldoc comments in multiple files

Also should the description of options be the same between observable and regular client concrete and interfaces?  Currently across the 4 places, there are 3 different wordings

> Options for changing the behaviour of the API

> Options to change the behaviour of the API

> Options to change the API response